### PR TITLE
Clarify createClient in README as per #781

### PR DESCRIPTION
**Pull request by stockholmux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,15 +144,17 @@ resume sending when you get `drain`.
...
 ### overloading
-* redis.createClient() = redis.createClient(6379, '127.0.0.1', {})
-* redis.createClient(options) = redis.createClient(6379, '127.0.0.1', options)
-* redis.createClient(unix_socket, options)
-* redis.createClient(port, host, options)
+* `redis.createClient(port,host,options)`
+* `redis.createClient()` is equivalent to `redis.createClient(6379, '127.0.0.1', {})`
+* `redis.createClient(options)` is equivalent to `redis.createClient(6379, '127.0.0.1', options)`
+* `redis.createClient(unix_socket, options)`
+* `redis.createClient(port, host, options)`
```
Comment (erinspice) [inline README.md]: Line 151 and 155 look identical.

File: README.md [hunk 2]
```diff
@@ -144,15 +144,17 @@ resume sending when you get `drain`.
...
 
-If you have `redis-server` running on the same computer as node, then the defaults for
-port and host are probably fine.  `options` in an object with the following possible properties:
+ `options` in an object with the following possible properties:
```
Comment (erinspice) [inline README.md]: Should be "options *is* an object".

**Comment by stockholmux** [comment]
fair enough

**Comment by erinspice** [comment]
@stockholmux Why did you close it? I would like to merge it in when those two tiny changes are made.

**Comment by stockholmux** [comment]
@erinspice Just did a another PR with the tiny changes - #783

**Pull request closed without merging**